angular/src/app/book-table/book-table.component.ts

```diff
@@ -34,78 +32,111 @@ export class BookTableComponent implements OnInit {
     invitedGuests: undefined,
   };
 
-  constructor(public window: WindowService,
-              public translate: TranslateService,
-              public snackBarservice: SnackBarService,
-              public dialog: MatDialog) {
-  }
+  constructor(
+    public window: WindowService,
+    public translate: TranslateService,
+    public snackBarservice: SnackBarService,
+    public dialog: MatDialog,
+  ) {}
 
   ngOnInit(): void {
     this.invitationForm = new FormGroup({
-      'bookingDate': new FormControl(this.reservationInfo.booking.bookingDate, Validators.required),
-      'name': new FormControl(this.reservationInfo.booking.name, Validators.required),
-      'email': new FormControl(this.reservationInfo.booking.email, [
+      bookingDate: new FormControl(
+        this.reservationInfo.booking.bookingDate,
+        Validators.required,
+      ),
+      name: new FormControl(
+        this.reservationInfo.booking.name,
         Validators.required,
-        Validators.pattern(/^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i),
+      ),
+      email: new FormControl(this.reservationInfo.booking.email, [
+        Validators.required,
+        Validators.pattern(
+          /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i,
+        ),
       ]),
-      'invitedGuests': new FormControl(this.invitationModel),
+      invitedGuests: new FormControl(this.invitationModel),
     });
 
     this.bookForm = new FormGroup({
-      'bookingDate': new FormControl(this.reservationInfo.booking.bookingDate, Validators.required),
-      'name': new FormControl(this.reservationInfo.booking.name, Validators.required),
-      'email': new FormControl(this.reservationInfo.booking.email, [
+      bookingDate: new FormControl(
+        this.reservationInfo.booking.bookingDate,
+        Validators.required,
+      ),
+      name: new FormControl(
+        this.reservationInfo.booking.name,
         Validators.required,
-        Validators.pattern(/^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i),
+      ),
+      email: new FormControl(this.reservationInfo.booking.email, [
+        Validators.required,
+        Validators.pattern(
+          /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i,
```

Comment:
Is this the same email pattern repeated twice? I know that not a part of your task, but will note it down to fix later

Reply:
Ok, perfect 👍